deploy/docker/Dockerfile-multi-arch-distroless

```diff
@@ -0,0 +1,36 @@
+# source https://github.com/istio/istio/blob/master/docker/Dockerfile.distroless
+FROM gcr.io/distroless/static-debian11:latest-${TARGETARCH:-amd64} AS distroless_source
+
+LABEL maintainer="kiali-dev@googlegroups.com"
+
+# prepare a base dev to modify file contents
+FROM ubuntu:focal as ubuntu_source
+
+# Modify contents of container
+COPY --from=distroless_source /etc/ /home/etc
+COPY --from=distroless_source /home/nonroot /home/nonroot
+RUN echo kiali:x:1000: >> /home/etc/group
+RUN echo kiali:x:1000:1000:/home/kiali:/sbin/nologin >> /home/etc/passwd
+
+# Customize distroless with the following:
+# - password file
+# - groups file
+# - /home/nonroot directory
+FROM distroless_source
+COPY --from=ubuntu_source /home/etc/passwd /etc/passwd
+COPY --from=ubuntu_source /home/etc/group /etc/group
+COPY --from=ubuntu_source /home/nonroot /home/nonroot
+
+ENV KIALI_HOME=/opt/kiali \
+    PATH=$KIALI_HOME:$PATH
+
+WORKDIR $KIALI_HOME
+
+ARG TARGETARCH
```

Comment:
This needs to be moved up to line 1 (before the first time TARGETARCH is used in this Dockerfile).  The way this is currently, only amd64 archs will work. Please see the ARG scope documentation to understand why you need to do this: https://docs.docker.com/engine/reference/builder/#scope

Reply:
hmmm... I just ran a test and it failed. The TARGETARCH wasn't set when I expected it to be. I'm looking into this myself to find out why moving the ARG up to line 1 doesn't work

Reply:
Ah... https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
```
The following ARG variables are set automatically:
...
TARGETARCH - architecture component of TARGETPLATFORM
```

Based on this, I guess we never have to specify "ARG TARGETARCH"